deploy/examples/cluster-on-pvc.yaml

```diff
@@ -28,7 +28,7 @@ spec:
     # size appropriate for monitor data will be used.
     volumeClaimTemplate:
       spec:
-        storageClassName: gp2
+        storageClassName: gp2-csi
```

Comment:
Are the changes in this file just for testing? Or perhaps it's time we updated them anyway.

Reply:
I think it's time to update this too XD